go/client/cmd_wot_list.go

```diff
@@ -63,13 +79,7 @@ func (c *cmdWotList) Run() error {
 	line := func(format string, args ...interface{}) {
 		dui.Printf(format+"\n", args...)
 	}
-	var targetUsername string
-	if c.ownWot {
-		targetUsername = c.G().Env.GetUsername().String()
-	} else {
-		targetUsername = *c.username
-	}
-	line("Web-Of-Trust for %s", targetUsername)
+	line("Web-Of-Trust for %s", *c.vouchee)
```

Comment:
if this can be `nil` (ewhen you specify all the vouches you have written without specifying a vouchee), this will panic

Reply:
so the CLI forces vouchee to always be specified (either to what is passed in, or to the logged-in username).
my general preference has been to explicate vouchee and voucher in calls, and not rely on nil arg behavior (even though the server supports it).

to summarize, currently the CLI only supports:
vouchee = { specified, me }
voucher = { nil, me }

Reply:
isn't this ticket to support fetching all of the vouches written by the fetcher?

Reply:
discussed in chat and updated